test/integration/targets/win_dns_record/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+- name: Clean slate
+  import_tasks: clean.yml
+  vars:
+    fail_on_missing: false
+
+- block:
```

Comment:
Trying to replicate the tests locally breaks for me as DNS feature is missing. This solve it to me:
~~~ powershell
   - name: check if feature is availble
     win_shell: if (Get-Command -Name Add-WindowsFeature -ErrorAction SilentlyContinue) { $true } else { $false }
     changed_when: False
     register: module_available

   - name: install DNS features
     when: module_available.stdout | trim | bool
     block:

       - name: ensure DNS services are installed
         win_feature:
           name: DNS
           state: present
         register: dns_install

~~~

Reply:
See https://github.com/johnboy2/ansible/pull/2